perceval/backends/core/askbot.py

```diff
@@ -73,11 +72,16 @@ def fetch(self, from_date=DEFAULT_DATETIME):
 
         :returns: a generator of items
         """
-
         if not from_date:
             from_date = DEFAULT_DATETIME
 
-        from_date = datetime_to_utc(from_date).timestamp()
+        kwargs = {'from_date': from_date}
+        return super().fetch("question", **kwargs)
```

Comment:
This is a matter of style but I'd rather return a variable instead the call to a method like this. It's better for debugging. Although, this is not the case because the returning value is an iterator. In any case, I think we should be consistent and return here a value.

```
questions = super().fetch("question", **kwargs)
	
return questions
```